generators/java/model/versions.yml

```diff
@@ -0,0 +1,125 @@
+- changelogEntry:
+  - summary: |
+      The Java Model generator is now on major version 1. To take this upgrade without any breaks, please add the below
+      configuration to your `generators.yml` file:
+      ```yaml
+      generators:
+        - name: fernapi/fern-java-spring
+          config:
+            disable-required-property-builder-checks: true
+      ```
+    type: break
+  - summary: |
+      Generated builder methods now enforce non-null checks for required fields, ensuring that all required 
+      fields are properly validated during object construction:
+        ```java
+        @java.lang.Override
+        @JsonSetter("name")
+        public NameStage name(@NotNull String name) {
+            this.name = Objects.requireNonNull(name, "name must not be null");
+            return this;
+        }
+        ```
+    type: improvement
+  createdAt: '2024-09-05'
+  irVersion: 46
```

Comment:
Is this IR version supposed to be less than the previous (i.e. `48`)? Maybe `48` should actually be `46`?

Reply:
Good catch, updated!